app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/MapEditionData/EventEditionTag.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sweng.eventmanager.repository.data.MapEditionData;
```

Comment:
package name --> mapEditionData instead

Reply:
moreover: repository.data package only holds data classes for stuff that is in firebase, mostly. The rest should probably not be here